oeps/architectural-decisions/oep-0004-arch-oauth-scopes.rst

```diff
@@ -1,11 +1,13 @@
+.. _OEP-4:
+
 OEP-4: Application Authorization (Scopes)
 #########################################
 
 .. list-table::
    :widths: 25 75
 
    * - OEP
-     - :doc:`OEP-4 <oep-0004-arch-oauth-scopes>`
+     - :ref:`OEP-4`
```

Comment:
Most of the time you've included the title in the ref name but here you didn't can we be consistent and use a label that has the oep number and a short title?